bwe/RateController.cpp

```diff
@@ -555,9 +680,9 @@ void RateController::onReportReceived(uint64_t timestamp,
             _model.queue.setBandwidth(currentEstimate - 0.1 * (currentEstimate - receiveRateKbps));
         }
     }
-    else if (!sameProbe && backlogReport.lossCount > 2 && backlogReport.lossRatio > 0.02 &&
+    else if (backlogReport.lossCount > 2 && backlogReport.lossRatio > 0.02 &&
```

Comment:
I think it is. As you said the lossRation should be low that `2%`. Then this should be just to ensure that a not good report with a very short period will not reduce the estimation just because it lost 2 packets and in that short period it represents more than 2% loss. Here we are ensure we only consider probes bigger that 150ms (where 3 losses represent 2%) or shorter probes if the lossCount is something significant (3 or more sound reasonable). For the long probes (more than 150ms) the condition is always more than 2% that also sounds very reasonable

Reply:
I think it is. > 2 is just to ensure we don't react because 1 or 2 loss in short probes. where it can represent more than 2%. In the long probes is expected to have a lot of packets. Then probably 3 packets less than 2% in the most of the cases.